Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -5328,6 +5328,22 @@ bool CSSPropertyParser::parseFontPaletteValuesDescriptor(CSSPropertyID propId)
 
 bool CSSPropertyParser::consumeFont(bool important)
 {
+    if (CSSPropertyParserHelpers::isSystemFontShorthand(m_range.peek().id())) {
+        auto systemFont = m_range.consumeIncludingWhitespace().id();
+        if (!m_range.atEnd())
+            return false;
+
+        // We can't store properties (weight, size, etc.) of the system font here,
+        // since those values can change (e.g. accessibility font sizes, or accessibility bold).
+        // Parsing (correctly) doesn't re-run in response to updateStyleAfterChangeInEnvironment().
+        // Instead, we store sentinel values, later replaced by environment-sensitive values
+        // inside Style::BuilderCustom and Style::BuilderConverter.
+        for (auto property : fontShorthand())
+            addProperty(property, CSSPropertyFont, CSSValuePool::singleton().createIdentifierValue(systemFont), important, true);
```

Comment:
Should probably remove the `true` here since that marks these as "implicit", whatever that means.

Reply:
The original point  was to represent whether values were explicit set or not, to include/exclude them from serialization, but the CSSOM spec has changed since, and now serialization should be the same regardless of whether a property was implicitly or explicitly set. So now it's mostly being used as a quick hack to makes longhands appear or disappear from serialization (because it's much faster than creating a custom function in StyleProperties). I think we should just get rid of that concept in the future for that reason.

As for now, it's fine to use it however is convenient to you.

Reply:
OK I will leave this alone for now.